core/diirt/diirt-features/pom.xml

```diff
@@ -30,7 +30,6 @@
         <!-- Plugin required to properly publish source feature -->
         <groupId>org.eclipse.tycho</groupId>
         <artifactId>tycho-p2-plugin</artifactId>
-        <version>${tycho.version}</version>
```

Comment:
OK, I'm doing other changes (providing an help page, ...). Next commit will have `<version>${tycho.version}</version>` back.
Thank you